package.json

```diff
@@ -29,15 +29,15 @@
     "url": "https://github.com/Pomax/react-onclickoutside/issues"
   },
   "scripts": {
-    "build": "npm run build:es && npm run build:commonjs",
+    "build": "run-p build:*",
     "build:es": "cross-env BABEL_ENV=es babel src -d es",
     "build:commonjs": "cross-env BABEL_ENV=commonjs babel src -d lib",
     "lint": "eslint src/*.js ./test",
     "test": "run-s test:**",
-    "test:basic": "run-s lint build",
+    "test:basic": "run-s lint build:commonjs",
```

Comment:
It would, however, test something we can't otherwise test: the conversion to old ES. While it is reasonable to assume babel has its own test suite, I've seen enough odd conversion quirks in the past to justify testing both just in case the original passes and the other fails.

I don't think it'll need any special affordances though: phantom should support modern JS... but as long as we file an issue for it we can deal with that on its own instead of in this PR